brave-sparkjava/src/main/java/com/github/kristofa/brave/sparkjava/BraveTracingSparkJava.java

```diff
@@ -0,0 +1,99 @@
+package com.github.kristofa.brave.sparkjava;
+
+import com.github.kristofa.brave.Brave;
+import com.github.kristofa.brave.ServerRequestInterceptor;
+import com.github.kristofa.brave.ServerResponseInterceptor;
+import com.github.kristofa.brave.ServerSpanThreadBinder;
+import com.github.kristofa.brave.ServerTracer;
+import com.github.kristofa.brave.http.DefaultSpanNameProvider;
+import com.github.kristofa.brave.http.HttpServerRequestAdapter;
+import com.github.kristofa.brave.http.HttpServerResponseAdapter;
+import com.github.kristofa.brave.http.SpanNameProvider;
+import com.github.kristofa.brave.servlet.internal.MaybeAddClientAddressFromRequest;
+import spark.ExceptionHandler;
+import spark.Filter;
+import spark.Request;
+import spark.Response;
+import zipkin.Constants;
+
+import static com.github.kristofa.brave.internal.Util.checkNotNull;
+
+public final class BraveTracingSparkJava {

```

Comment:
I feel the SparkJava suffix is implied. The intent with the `BraveTracing` name was that in sparkjava stack traces or debug, you'd see "brave and tracing" there.

SparkJava is redundant as all the types returned are defined in sparkjava.

If we adopt a more verbose naming convention, then it will make other things a bit longer, too. Ex in okhttp as opposed to BraveTracingInterceptor, it would be BraveTracingOkHttpInterceptor (eventhough it would only be usable in okhttp due to the type being declared in okhttp)

I know naming conventions are hard, but hoping you might agree that dropping the SparkJava suffix doesn't hurt and also makes for a more consistent and less verbose naming strategy for us to apply to other code.